src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```
@@ -1075,6 +1075,46 @@ properties:
             type: integer
             fits_keyword: DETYSIZ
             blend_table: True
+          multistripe_reads1:
+            title: Num. substripe rows read starting at SUBSTRT1
+            type: number
+            fits_keyword: MSTR_RD1
+            blend_table: True
+          multistripe_skips1:
+            title: Num. rows to skip after reading MSTR_RD1 rows
+            type: number
```

```suggestion
          multistripe_skips1:
            title: Number of rows to skip after reading MSTR_RD1 rows
            type: number
```
Same as above, should this be an integer?